packages/data-context/test/unit/sources/MigrationDataSource.spec.ts

```diff
@@ -0,0 +1,69 @@
+import { expect } from 'chai'
+import sinon from 'sinon'
+
+import { DataContext } from '../../../src'
+import { MigrationDataSource } from '../../../src/sources'
+import { createTestDataContext } from '../helper'
+
+const pkg = require('@packages/root')
+
+describe('MigrationDataSource', () => {
+  context('.migration', () => {
+    let ctx: DataContext
+    let fetchStub: sinon.SinonStub
+    let migrationDataSource: MigrationDataSource
```

Comment:
Looks like `migrationDataSource` could be a const within each test, since it's not shared between hook -> test body like the other two variables here.